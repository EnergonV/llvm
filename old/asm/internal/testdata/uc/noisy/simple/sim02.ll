; ModuleID = 'sim02.c'
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

@foo = common global i32 0, align 4

; Function Attrs: nounwind uwtable
define i32 @main() #0 {
  %bar = alloca i32, align 4
  store i32 76, i32* @foo, align 4
  store i32 54321, i32* %bar, align 4
  %1 = load i32, i32* @foo, align 4
  call void @putint(i32 %1)
  %2 = load i32, i32* %bar, align 4
  call void @putint(i32 %2)
  ret i32 0
}

declare void @putint(i32) #1

attributes #0 = { nounwind uwtable "disable-tail-calls"="false" "less-precise-fpmad"="false" "no-frame-pointer-elim"="true" "no-frame-pointer-elim-non-leaf" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+sse,+sse2" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #1 = { "disable-tail-calls"="false" "less-precise-fpmad"="false" "no-frame-pointer-elim"="true" "no-frame-pointer-elim-non-leaf" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+sse,+sse2" "unsafe-fp-math"="false" "use-soft-float"="false" }

!llvm.ident = !{!0}

!0 = !{!"clang version 3.7.1 (tags/RELEASE_371/final)"}
