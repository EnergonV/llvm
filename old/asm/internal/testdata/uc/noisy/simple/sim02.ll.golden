target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"
@foo = global i32 0
define i32 @main() {
0:
	%bar = alloca i32
	store i32 76, i32* @foo
	store i32 54321, i32* %bar
	%1 = load i32, i32* @foo
	call void @putint(i32 %1)
	%2 = load i32, i32* %bar
	call void @putint(i32 %2)
	ret i32 0
}
declare void @putint(i32)
